src/actions/org.js

```diff
@@ -52,7 +52,7 @@ export const resetFileDisplay = () => {
 };
 
 const getDebouncedSyncFunction = () =>
-  debounce((dispatch, options) => dispatch(doSync(options)), 3000, {
+  debounce((dispatch, options) => dispatch(doSync(options)), 15000, {
```

Comment:
This is only tangentially related to adding GitLab as a sync backend, but the original 3 second delay seemed really aggressive to me. This is especially true for a git-based backend where every change results in a new commit, but this also decreases network traffic and battery drain regardless of backend. I think 15 seconds is still short enough to be imperceptible to the user considering how long it takes to e.g. put your phone down and refresh a file on your computer.

If this is a bad idea I can revert it.

GitLab's API [rate limits to 2,000 requests/minute for authenticated users](https://about.gitlab.com/blog/2020/12/17/automation-check-in-and-rate-limit-changes-on-gitlab-dot-com/), so that isn't a concern.

Reply:
Do I get it right, that this constant is the interval to check for changes and push?

I think Organice checks only, if it's in foreground and it only makes API calls if there are changes? If so, a reason for a short interval is that it's more likely that Organice pushes my changes automatically before I switch the phone display off. However, if Organice fails to push automatically due to a longer interval, and I forget to trigger it, it may lead to data-loss. Just my thoughts.

Reply:
> This is only tangentially related to adding GitLab as a sync backend, but the original 3 second delay seemed really aggressive to me.

It is supposed to be aggressive. When running organice on a smartphone, it is very important to quickly dispatch the latest changes to the sync-backend before the user puts organice into the background or shuts off the screen. The reason is that smartphone OSs will give the application only a _very short_ time to get things in order.

I understand that on a git-based back-end you might prefer to control how many commits are created. For the reason of controllings the time of synchronisation, we have these two settings that you can switch off:

![image](https://user-images.githubusercontent.com/505721/139580430-149b68f0-9e49-474d-9a59-e393476eaa9a.png)

Reply:
Makes sense, I dropped the commit for this change so it's back to the original 3 seconds.

Reply:
Perfect, thanks!